### PR TITLE
Diagram fix

### PR DESCRIPTION
**Pull request by g2flyer**
**What this PR does / why we need it**:

Fixed protocol inconsistency/bug in touchpoint diagram while going through RFC.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing changes and/or breaks backward compatability?**:

```

### CODE REVIEW

File: docs/design/fabric-v2+/fpc-integration-touchpoints.puml [hunk 1]
```diff
@@ -63,19 +64,33 @@ group Per FPC chaincode FCC
...
+   & Peer1 <-> ecc1 
    activate ecc1
-   activate ecc2
+
+   alt "full" FPC (Post-MVP)
+      Client2 -> Peer2 : chaincode initEnclave FPC
+      & Peer2 <-> ecc2 
+      Client1 -> Peer1 : chaincode exportCCKeys FPC
+      & Peer1 <-> ecc1 
+      activate ecc2
```
Comment (mbrandenburger) [inline docs/design/fabric-v2+/fpc-integration-touchpoints.puml:90]: moving `activate ecc2` after `& Peer2 <-> ecc2 ` would look a bit better.

**Review by mbrandenburger — APPROVED** [review]
Thank Michael! LGTM. Just a few minor improvements below.

**Comment by mbrandenburger** [comment]
@bvavala please have a quick look and merge if you are also happy!

File: docs/design/fabric-v2+/fpc-integration-touchpoints.puml [hunk 2]
```diff
@@ -63,19 +64,33 @@ group Per FPC chaincode FCC
...
 
    Client1 -> Peer1 : chaincode approveformyorg FPC
-   Client1 -> Peer2 : chaincode approveformyorg FPC
+   Client2 -> Peer2 : chaincode approveformyorg FPC
 
    Client1 -> Peer1 : chaincode commit FPC
    note over Peer1
      trigger preparation of execution enclaves & key-distribute for all peers
    end note
    Peer1 <-> Peer2
```
[the remark this answers is not in the document]
Comment (g2flyer) [reply docs/design/fabric-v2+/fpc-integration-touchpoints.puml:79]: yep, note definitely is wrong. there is actually some involvement of Peer2 via orderer. But we don't do it for approvaeformyorg, so seems best to skip it also here (the alternative of mapping all the flows involving orderer seem to messy to cover ...

File: docs/design/fabric-v2+/fpc-integration-touchpoints.puml (hunk 1 above)
Comment (g2flyer) [reply docs/design/fabric-v2+/fpc-integration-touchpoints.puml:90]: yep, was too late, moved it up ..

**Pull request merged**